r-pkg/R/elasticsearch_parsers.R

```diff
@@ -1104,7 +1104,7 @@ es_search <- function(es_host
     }
     
     # Make request
-    result <- httr::POST(url = reqURL, body = query_body)
+    result <- httr::RETRY(url = reqURL, body = query_body)
```

Comment:
Thank you for submitting a PR! However, this still needs some work:

1) See `?RETRY` for how that function works. They're not literally 1-to-1 swappable like this. You'll need to set some value for number of retries and provide an HTTP verb. You can test by trying to hit `"http://www.google.com"` with `httr::GET` and then seeing if you can get the same result with `httr::RETRY()`

2) Please add `RETRY` to the [roxygen docs](https://cran.r-project.org/web/packages/roxygen2/vignettes/roxygen2.html) for this function and regenerate them using `devtools::document`. You should see at least one `.Rd` file show up in the diff

3) This is not the only occurrence of `httr::POST` in the package. There are actually 4 of them. You can find them with `git grep` or with [find in files](https://support.rstudio.com/hc/en-us/articles/200710523-Navigating-Code) if you have RStudio.